base/distributed/managers.jl

```diff
@@ -455,10 +455,10 @@ end
 const client_port = Ref{Cushort}(0)
 
 function socket_reuse_port()
-    @static if is_linux() || is_apple()
+    if Bool(ccall(:jl_has_so_reuseport, Int32, ()))
```

Comment:
Realized that this can be `ccall(:jl_has_so_reuseport, Bool, ())`. I'll make that change if this approach is acceptable.